index.html

```diff
@@ -383,7 +383,6 @@ <h1 class="project-name">neveragain.tech</h1>
 <li>Meghan Purvis <!--from github user meghanpurvis -->
 <li>Melinda Jacobs
 <li><a href="https://twitter.com/0xabad1dea">Melissa Elliott</a>
-<li><a href="https://twitter.com/mcohen01">Michael Cohen, Twilio</a> <!-- from github user mcohen01 -->
```

Comment:
It's showing that I deleted this line, for some reason. Not intentional. I assume there is a Git Gremlin at work.